nilearn/signal.py

```diff
@@ -26,20 +26,24 @@ def _standardize(signals, detrend=False, normalize=True):
 
     Parameters
     ==========
-    signals: numpy.ndarray
+    signals : numpy.ndarray
```

Comment:
Here is an image of generated doc. For the first parameter, there is no space. For the second one, I put a space. You can see that the extra space shows in the doc. This is, AFAIK, why we decided not to put them (this is the same in scikit-learn). I am not against adding them but we should be consistent and do that in a dedicated PR.

![space](https://cloud.githubusercontent.com/assets/1647301/7248651/a5291bba-e814-11e4-8c76-9ae3ecb2d685.png)